_studio/mfx_lib/encode_hw/h265/src/mfx_h265_encode_hw.cpp

```diff
@@ -1105,6 +1113,24 @@ void SetEncFrameInfo(MfxVideoParam &m_vpar,
     }
 }
 
+inline mfxI8 GetMinQP(MfxVideoParam &vpar)
+{
+    return (mfxI8)((IsOn(vpar.mfx.LowPower) || (vpar.m_platform >= MFX_HW_KBL
+#if (MFX_VERSION >= 1025)
+        && vpar.m_platform < MFX_HW_CNL
+#endif
+        )) ? 0 : -6* vpar.m_sps.bit_depth_luma_minus8);
```

Comment:
why 6? Can this be used as some macro or const int with the reasonable name instead of magic number?

Reply:
constant from standard (no name)